_content/language-and-grammar.md

```diff
@@ -58,47 +59,52 @@ If a sentence ends with an abbreviation, the period used for the abbreviation al
 For more information, contact Agora Inc.
 :::
 
+#### File extensions
+
+When you refer to file extensions, use all caps and do not use a period before.
```

Comment:
I think filename extensions do not equal to file format types. I feel it's ok to say something like "files with the `.jpg` extension".

Reply:
So maybe this title should be File format types? Also, in some cases, the format name can be lowercase, for example, a text file.